cpssEnabler/linuxNoKernelModule/drivers/dmaDriver.c

```diff
@@ -942,10 +945,8 @@ static loff_t mvDmaDrv_lseek(struct file *file, loff_t off, int unused)
 	if (platdrv_dev) /* device-tree reservation */
 		return 0;
 
-	m->pci_offset = off;
 	pdev = pci_get_domain_bus_and_slot(domain, bus, devfn);
```

Comment:
Why were pci_offset and pdev initialization removed?

Reply:
@eladnachman - It has been put back, inadvertently removed as it was not there in cpss internal code.